vic/drivers/image/Makefile

```diff
@@ -38,6 +38,18 @@ SHAREDIMAGEPATH = ../shared_image
 # VIC RUN PATH
 VICPATH = ../../vic_run
 
+# VIC EXTENTIONS PATH
+EXTPATH = ../../extensions
+
+# Set the routing model extension
+# | Level      | Value                              |
+# |----------  |---------------------------------   |
+# | rout_stub  | Stub routing model (no routing)    |
+# | rout_rvic  | Use RVIC routing scheme            |
+ROUT=rout_stub
```

Comment:
should this be  `ROUT := rout_stub` to allow it to be set on the command line via `make ROUT=rout_rvic`?

Reply:
ROUT=rout_rvic is also working with ROUT=rout_stub, but for future additions it would probably better to use your suggestion. We included it.